plugins/modules/ios_bgp_address_family.py

```diff
@@ -547,11 +547,17 @@
                     description: Only give warning message when limit is exceeded
                     type: bool
               next_hop_self:
+                description:
+                  - Disable the next hop calculation for this neighbor
+                  - This option is DEPRECATED and is replaced with nexthop_self which
```

Comment:
Can you also mention that it will be removed in a release after 2023-06-22?

Reply:
@NilashishC  Thanks